src/screens/WebScreen.js

```diff
@@ -42,7 +42,7 @@ export const WebScreen = ({ route }) => {
         // https://github.com/react-native-webview/react-native-webview/blob/19980d888d66554875f3ac64b3e8a35bd7ad998b/src/WebViewTypes.ts#L378-L389
         decelerationRate="normal"
         // https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#applicationnameforuseragent
-        applicationNameForUserAgent={appJson.expo.scheme}
+        applicationNameForUserAgent={!webUrl.includes('bbnavi') ? appJson.expo.scheme : ''}
```

Comment:
this prop is not required, so I would suggest to set `undefined` when not used instead of an empty string

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/051c46c54eabbbea189e34caecce23c33ad76bd3